### PR TITLE
@dzucconi: Can query sale artworks by `ids`

### PR DESCRIPTION
**Pull request by craigspaeth**
With my latest two Gravity PRs, this gets us most of the way to being able to get data for the My Active Bids UI.

Also minor things looped in:

* Added LICENCE file
* Re-arranged .env vars to be obvious what needs to change for development

Killer job on Metaphysics! Very easy dev experience.

### CODE REVIEW

File: schema/sale_artworks.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+} from 'graphql';
+
+const SaleArtworks = {
+  type: new GraphQLList(SaleArtwork.type),
+  description: 'A list of Sale Artworks',
+  args: {
+    ids: {
+      type: new GraphQLList(GraphQLString),
+      description: 'A list of sale artwork ids',
+    },
```
Comment (dzucconi) [inline schema/sale_artworks.js]: I think this is fine and probably easier to consume, but you also don't *really* have to expose an explicit schema for this if you can generate the appropriate query:

```
{
  my_first_sale_artwork: sale_artwork(id: "my_first_sale_artwork") { ... }
  my_second_sale_artwork: sale_artwork(id: "my_second_sale_artwork") { ... }
  # etc...
}
```

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ It is currently used in production from the `/artists/` route on [artsy.net](htt
 
 ### Getting Setup
 
-Set up your `.env` file based on our example `.env.test`.
+Set up your `.env` file based on our example `.env.test` (the variables you must change for development are separated at the bottom).
```
Comment (dzucconi) [inline README.md]: :+1:

**Comment by dzucconi** [comment]
:+1: :+1:

File: schema/sale_artworks.js (hunk 1 above)
Comment (craigspaeth) [inline schema/sale_artworks.js]: I'm definitely cool with the composed query you show

**Pull request merged**